sky/server/requests/process.py

```diff
@@ -67,14 +68,14 @@ def shutdown(self, wait: bool = True) -> None:
 
 
 # Define the worker function outside of the class to avoid pickling self
-def _disposable_worker(fn, initializer: Optional[Callable], initargs: Tuple,
-                       args, kwargs):
+def _disposable_worker(fn, initializer, initargs, result_queue, args, kwargs):
```

Comment:
Can we explain what these args are in a comment? I was very confused about result_queue for a bit.

Reply:
Sure!